cvat-core/src/api-implementation.ts

```diff
@@ -117,6 +119,11 @@ export default function implementAPI(cvat) {
         return result;
     };
 
+    cvat.server.requestAll.implementation = async (url) => {
+        const result = await serverProxy.server.requestAll(url);
+        return result;
+    };
+
```

Comment:
I believe we do not need extra method for this action. 
Better to add some arguments to the existing method